### PR TITLE
widen before searching current hunk #88

### PR DESCRIPTION
**Pull request by syohex**
This is related to #88

### CODE REVIEW

File: git-gutter.el [hunk 1]
```diff
@@ -664,12 +664,14 @@ gutter information of other windows."
...
 
 (defun git-gutter:search-here-diffinfo (diffinfos)
-  (cl-loop with current-line = (line-number-at-pos)
-           for diffinfo in diffinfos
-           for start = (plist-get diffinfo :start-line)
-           for end   = (or (plist-get diffinfo :end-line) (1+ start))
-           when (and (>= current-line start) (<= current-line end))
-           return diffinfo))
+  (save-excursion
+    (widen)
```
Comment (xuchunyang) [inline git-gutter.el]: widen needs save-restriction, not save-excursion, otherwise, user's narrowing will be removed.

Comment (xuchunyang) [inline git-gutter.el]: Works fine when replace save-excursion with save-restriction.

Comment (syohex) [inline git-gutter.el]: Sorry I mistook.

Comment (syohex) [inline git-gutter.el]: I fixed at #90

**Comment by xuchunyang** [comment]
Thanks.  Closing #88

**Pull request merged**